lib/template/tests/test_template.c

```diff
@@ -425,6 +425,10 @@ Test(template, test_escaping)
                                        FALSE, "\"value\"");
   assert_template_format_with_escaping("$(if (\"${APP.VALUE}\" eq \"value\") \"${APP.QVALUE}\" \"${APP.QVALUE}\")",
                                        TRUE, "\\\"value\\\"");
+
+  /* literal strings are not escaped */
+  assert_template_format_with_escaping("\"$(echo $(echo $(length ${APP.QVALUE})))\"", TRUE, "\"7\"");
```

Comment:
This is not a literal string, so I am not sure the comment matches the testcase.

Reply:
I agree, this comment is misleading. What I wanted to check with these is that if there are tf/macro parts of the template, they are escaped, but the literal string parts are not (`"` in the beginning and the end).

I will rephrase this comment.

Reply:
`/* literal parts of the template are not escaped */`

Reply:
:) thanks